worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java

```diff
@@ -34,6 +34,29 @@
 
 /**
  * An axis-aligned cuboid. It can be defined using two corners of the cuboid.
+ *
+ * Example usage:
+ * <pre>
+ * <code>
+ * Bukkit.getScheduler().runTaskTimerAsynchronously(this, () -> {
```

Comment:
Why are you using platform-specific code in the example usage here, let alone async code which is not safe for alterations to the world